lib/srfi/%3a176/version-flag.sls

```diff
@@ -0,0 +1,16 @@
+(library (srfi :176 version-flag)
+         (export version-alist)
+         (import (rnrs) (only (mosh config) get-config))
+
+(define (version-alist)
+  `((version ,(get-config "version"))
+      (command "mosh")
+      (scheme.id mosh)
+      (install-dir ,(string-append (get-config "prefix") "/bin"))
+      (languages scheme r6rs)
+      (encodings utf-8)
+      (cache-path ,(get-config "mosh-cache-dir"))
```

Comment:
Non-standard options (not mentioned in the SRFI) should either:

* Be prefixed with the implementation's `scheme-id`, i.e. `(mosh.cache-path ,(get-config "mosh-cache-dir"))`
* Be added to `https://registry.scheme.org/#version-flag-property`.

Is the Mosh cache-path for byte-compiled files?

Reply:
Fixed.

> Is the Mosh cache-path for byte-compiled files?

Yes.